composer.json

```diff
@@ -13,29 +13,33 @@
         "antimattr/mongodb-migrations" : "*"
     },
     "require": {
-        "php": ">=5.4.45",
-        "doctrine/mongodb": "1.*",
-        "symfony/console": "^2.5|^3.0|^4",
-        "symfony/yaml": "^2.5|^3.0|^4"
+        "php": "^5.6|^7.1",
+        "doctrine/mongodb": "^1.0",
+        "symfony/console": "^3.3|^4",
```

Comment:
Why we drop symfony 2.7 and 2.8 and keep 3.3 ? AFAIK 2.7 and 2.8 are still supported.

Reply:
tbh I saw 2.7 support finishes in May and thought that we don't maintain this library much so might as well drop it - but I had forgotten about 2.8. So I'll restore `^2`